content/blog/code-deploy-and-manage-a-serverless-rest-api-on-aws-with-pulumi/index.md

```diff
@@ -58,13 +55,12 @@ If you're on Windows, run this:
     SET "PATH=%PATH%;%USERPROFILE%.pulumiin"
 
 You'll deploy this app to your own AWS account, so follow the steps to
-[configure your AWS account](https://pulumi.io/install/aws.html).
+[configure your AWS account]({{< ref "/docs/quickstart/aws" >}}).
```

Comment:
Same. This should probably link to the page on actually getting the AWS account setup, which is now `/docs/reference/clouds/aws/setup.md`